bpf/bpf_lxc.c

```diff
@@ -905,6 +905,8 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label, __u8 *reason,
 		 */
 		send_trace_notify6(ctx, TRACE_TO_PROXY, src_label, SECLABEL, &orig_sip,
 				  0, ifindex, 0, monitor);
+		if (tuple_out)
+			*tuple_out = tuple;
```

Comment:
nit: any reason to not use optimised memcpy we have?

Reply:
No reason, makes sense to fix that up.